Expanded beer categories and styles

nicolaspe opened the pull request:
Used the 2015 BJCP (Beer Judge Certification Program) Style Guidelines. It has 34 overall categories with different specific styles under each one.
Link to the guidelines: https://www.bjcp.org/docs/2015_Guidelines_Beer.pdf

nicolaspe committed:
Expanded beer categories and styles

Used the 2015 BJCP (Beer Judge Certification Program) Style Guidelines. It has 34 overall categories with different specific styles under each one.
Link to the guidelines: https://www.bjcp.org/docs/2015_Guidelines_Beer.pdf

nicolaspe committed:
formatting update

removed a final ',' that made the file not pass the checks

shiffman commented on data/foods/beer_categories.json:
```diff
@@ -1,16 +1,630 @@
 {
-  "description": "A list of beer categories.",
+  "description": "A list of beer categories and corresponding styles, according to the BJCP 2015 Style Guidelines",
```
I would include a `"source"` property as well with a link to the PDF you referenced!

nicolaspe replied:
added!

shiffman reviewed (commented):
Great job! I wonder if the maintainers of corpora would prefer to have both a simple array of beer categories and this more complex and detailed dataset? I would probably do what you did and expand the original file, but just curious to ask the question!

nicolaspe committed:
add source

Add source to the json object linking to the BJCP 2015 Style Guidelines pdf file

nicolaspe committed:
removing extra note from @shiffman branch
